modules/aws/ec2-files.go

```diff
@@ -221,18 +221,18 @@ func FetchFilesFromAsgsE(t *testing.T, awsRegion string, spec RemoteFileSpecific
 				return err
```

Comment:
This feels inconsistent to me. Below, when I encounter errors I add them to an array, but here I bounce out. I'm going to change behavior such that we try to scp files from however many nodes we can. Then return all the errors we encountered along the way.

Reply:
Updated the logic to be more consistent as I described above.